src/clarity-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -20,12 +20,12 @@ import {IfOpenService} from "../../utils/conditional/if-open.service";
 })
 export class DropdownItem {
 
-    constructor(private _dropdown: Dropdown, private el: ElementRef, private ifOpenService: IfOpenService) {
+    constructor(private dropdown: Dropdown, private el: ElementRef, private ifOpenService: IfOpenService) {
     }
 
     @HostListener("click")
     onDropdownItemClick(): void {
-        if (this._dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
+        if (this.dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
```

Comment:
i know this was there before but still :-( do you think our recommendation should be that users should just remove the disabled links or change disabled links with a label with disabled styling but this is invalid html.